docs/networking.adoc

```diff
@@ -0,0 +1,766 @@
+oxControlService12 02:08:20:08:ad:5d  dynamic REACHABLE    fd00:1122:3344:104::22     
+oxControlService12 02:08:20:c6:45:43  dynamic REACHABLE    fd00:1122:3344:104::23     
+oxControlService12 02:08:20:05:1e:b3  dynamic REACHABLE    fd00:1122:3344:104::24     
+oxControlService12 02:08:20:a3:a9:b7  dynamic REACHABLE    fd00:1122:3344:104::25     
+oxControlService12 02:08:20:ec:25:f9  dynamic REACHABLE    fd00:1122:3344:104::26     
+oxControlService12 02:08:20:8c:12:c2  dynamic REACHABLE    fd00:1122:3344:104::27     
+oxControlService12 02:08:20:58:f2:54  dynamic REACHABLE    fd00:1122:3344:104::29     
+oxControlService12 02:08:20:32:ba:8d  dynamic REACHABLE    fd00:1122:3344:104::2a     
+oxControlService12 02:08:20:57:00:a9  local   REACHABLE    fe80::8:20ff:fe57:a9       
+oxControlService12 33:33:ff:57:00:a9  other   REACHABLE    ff02::1:ff57:a9            
+----
+
+We see that `fd00:1122:3344:104::22` is there with MAC address `02:08:20:08:ad:5d`.  So Nexus can send packets directly to `fd00:1122:3344:104::22`.  The L2 frame will have MAC addr `fd00:1122:3344:104::22`.  Where will it go?  Let's see the routing table:
+
+[source,console]
+----
+root@oxz_nexus_c4c40c19-60de-4c8b-b201-2a367d8aa322:~# netstat -rn -f inet6
+
+Routing Table: IPv6
+  Destination/Mask            Gateway                   Flags Ref   Use    If
+--------------------------- --------------------------- ----- --- ------- -----
+::1                         ::1                         UH      2       0 lo0
+fd00:1122:3344:104::/64     fd00:1122:3344:104::4       U      11    2386 oxControlService12
+fe80::/10                   fe80::8:20ff:fe57:a9        U       2       0 oxControlService12
+default                     fd00:1122:3344:104::1       UG     13  210937
+----
+
+Traffic aimed at `fd00:1122:3344:104::22` will match the rule for `fd00:1122:3344:104::/64`.  The fact that the gateway address is in this zone and there's an interface in the rule (`oxControlService12`) means that we're directly connected to hosts in this prefix and traffic is sent directly out the `oxControlService12` VNIC.  The packet reaches the etherstub, which acts as a virtual switch and so knows that MAC `02:08:20:08:ad:5d` is on `oxControlService1` and delivers it to that VNIC -- in the CockroachDB zone.  Hooray!
+
+We can summarize the underlay networking configuration on a typical sled like this:
+
+```mermaid
+graph TD;
+    Sidecar0
+    Sidecar1
+    cxgbe0["cxgbe0 (phys)"]
+    cxgbe1["cxgbe1 (phys)"]
+    cxgbe0-- "backplane" --- Sidecar0
+    cxgbe1-- "backplane"  --- Sidecar1
+    cxgbe0_ip["cxgbe0 (IP, GZ)\naddresses:\nLL: fe80::..."]
+    cxgbe1_ip["cxgbe1 (IP, GZ)\naddresses:\nLL: fe80::..."]
+    cxgbe0_ip --- cxgbe0
+    cxgbe1_ip --- cxgbe1
+
+    underlay0["underlay0 (VNIC, GZ)"]
+    underlay0_ip["underlay0 (IP, GZ)\naddresses:\nLL: fe80::..."]
+
+    subgraph cpzone["Every control plane zone\n(Nexus, CockroachDB,\nDNS, etc. and oxz_switch)"]
```

Comment:
The bit in parens is cut off in the github rendering, fwiw.

Reply:
Yeah...I just removed this.